README.md

```diff
@@ -20,15 +20,33 @@
 
 # Prefect
 
-Prefect is an orchestrator for data-intensive workflows. It's the simplest way to transform any Python function into a unit of work that can be observed and orchestrated. With Prefect, you can build resilient, dynamic workflows that react to the world around them and recover from unexpected changes. With just a few decorators, Prefect supercharges your code with features like automatic retries, distributed execution, scheduling, caching, and much more. Every activity is tracked and can be monitored with the Prefect server or Prefect Cloud dashboard.
+Prefect is a workflow application framework. It's the simplest way to transform Python code into an interactive workflow application that can be observed and orchestrated.
```

Comment:
@discdiver  How about "Prefect is an orchestration and observability platform for building, observing, and triaging workflows. It's the simplest way to transform Python code into an interactive workflow application."